src/Quartz.Tests.Unit/Collections/OrderedDictionaryTest.cs

```diff
@@ -0,0 +1,36 @@
+using NUnit.Framework;
+using Quartz.Collections;
+using System;
+
+namespace Quartz.Tests.Unit.Collections
```

Comment:
For new code can use file-scoped namespaces

Reply:
Indeed, it's still growing on me. Other "new" C# features like **using declarations** will always get lost while travelling the distance between my brain and my keyboard :p